### PR TITLE
pressure jump outline

### PR DESCRIPTION
**Pull request by stan-dot**
not sure about the wiring

### CODE REVIEW

File: src/i22_bluesky/plans/pressure_jump.py [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+        rows.append(
+            SeqTableRow(
+                repeats=pre_jump_frames,
+                time1=in_micros(exposure),
+                outa1=True,
+                outb1=True,
+                time2=in_micros(deadtime),
+                outa2=True,
+            )
+        )
```
Comment (DiamondJoseph) [inline src/i22_bluesky/plans/pressure_jump.py:329]: row for the triggering , hold the pressure cell trigger high until end of experiment

**Comment by stan-dot** [comment]
todo: update with notes from the meeting

**Comment by stan-dot** [comment]
1, 3 pressure must be less than 50 bar. for rise 5 mst be open, for loss 6 must be open

transducer aka chamber
p1 pump, p2 intermediate, p3 sample
static scanspec

read for cell - 3

pump is fast


configuration is jsut 11 PVs; 3 per valbe, and fast ones as extra


on joining the volumes, the pressure would fall

we do not want any air, it would cause the diagram to wobble and various issues



the physical orde ris pump - v3 - inter -> v5/v6 - celll

match inter pressure cell pressure, then just raise the pump

there are some plugins

in GDA there is pressure_from, pressure_to, etc, and old style of TFG table "1110111" etc binary string

**Comment by stan-dot** [comment]
todos
- start the plan with the confirmation of values
> 1, 3 pressure must be less than 50 bar. for rise 5 mst be open, for loss 6 must be open

- make a stub just to adjust those, add it to a third plan
- just increase the pressure and assert that it's higher at the end
- ignore the detectors for the small test

File: src/i22_bluesky/plans/pressure_jump.py (hunk 1 above)
Comment (stan-dot) [reply src/i22_bluesky/plans/pressure_jump.py:285]: @DiamondJoseph any updates?

**Comment by stan-dot** [comment]
ideally should take into account this issue too https://github.com/DiamondLightSource/dodal/issues/759

File: src/i22_bluesky/plans/pressure_jump.py (hunk 1 above)
Comment (stan-dot) [reply src/i22_bluesky/plans/pressure_jump.py:285]: actually all the outs are set to True in all the other frames

**Pull request merged**